doc/iterations/2025.2/add_project_elements_selection_to_urls.adoc

```diff
@@ -0,0 +1,89 @@
+
+But there are no URLs that can be used to designate a selection of *project elements* (_models_ and/or _representations_ and/or _semantic elements_).
+This possibility would be useful for documentation and collaboration purposes.
+
+== Key Result
+
+End users shall be able to create links that include a selection of project elements.
+
+=== Acceptance Criteria
+
+- End users can easily obtain a URL from their current selection of project elements in the workbench.
+- When resolving a URL that specifies a selection, the specified project elements automatically get selected in the workbench.
+
+== Solution
+
+- The current URL will change based on the selection made in the workbench.
+- The identifiers of the selected project elements will be stored in the URL in parameter _selection_.
+- The order of the project element identifiers in the URL reflects the order of the project elements in the selection.
+- A project element may appear several times in the selection.
+  This is for instance the case if several different graphical elements in a representation represent the same semantic element.
+- Upon resolving a URL with parameter _selection_, the workbench selection is set to the specified project elements.
+The resulting behavior may depend largely on how the workbench and its sections decide to implement this behavior.
+- In the default workbench, the selection will be dispatched to the _Representation Editor_ (if opened), or otherwise to the _Explorer_ view.
+- In both the _Explorer_ view and the _Representation Editor_, selecting a semantic element means selecting all the graphical elements whose _target_ is that semantic element.
+In particular, this means there may be a lot of graphical elements that end up selected, because representations may include any number of graphical elements to represent a single semantic element.
+
+=== Scenario 1 : Obtaining the URL from a workbench selection
+
+- The end user is in a project.
+- If no representation is opened, the URL looks like `+https://example.org/projects/project-id/edit+`.
+- If a representation is opened, the URL looks like `+https://example.org/projects/project-id/edit/representation-id+`.
+
+The following behavior is the same in both cases.
+
+- The end user selects project elements.
+This can be done through any section of the workbench that supports selections: _Explorer_ view, _Representation Editor_, _Related Elements_ view, etc.
+
+- The selected elements can be:
+-- _Models_, e.g. the root elements in the _Explorer_ view.
+-- _Representations_, e.g. the _Explorer_ view displays them below their target element.
+-- _Semantic elements_, contained by a _model_ or a _semantic element_.
+-- _Graphical elements_, i.e. part of a representation.
+In that case, the intention is to capture in the selection their target semantic elements.
```

Comment:
Technically we don't know, on purpose, what a selected element is. It's not limited to those four use cases, we just don't know nor care.